docs/modules.rst

```diff
@@ -1,7 +0,0 @@
-jax
-===
-
-.. toctree::
-   :maxdepth: 4
-
-   jax
```

Comment:
Was dropping this file intentional?

Reply:
It was. I could not find any references to this file, and in fact there were warnings as such. I think that the role this was was supposed to play is played by the reference to "jax.rst" in index.rst.